pywinauto/hooks.py

```diff
@@ -0,0 +1,231 @@
+from ctypes import wintypes
+from ctypes import windll, CFUNCTYPE, POINTER, c_int, c_void_p, byref
+import atexit
+
+
+def create_pointer(handler):
+    cmp_func = CFUNCTYPE(c_int, c_int, c_int, POINTER(c_void_p))
+    return cmp_func(handler)
+
+
+class KeyboardEvent:
```

Comment:
New-style classes are recommended to use everywhere, i.e. just inherit all classes from `object`.